content/en/docs/tasks/virtual-machines/multi-network/index.md

```diff
@@ -1,8 +1,14 @@
 ---
-title: Multi-network Mesh Expansion
-description: Integrate VMs and bare metal hosts into an Istio mesh deployed on Kubernetes with gateways.
+title: Virtual Machines in Multi-network Meshes
+description: Work with virtual machines and bare metal hosts within your Istio mesh deployed across multiple networks on Kubernetes.
```

Comment:
This description isn't quite right either. It's showing how to add a service running on a VM in a different network to a mesh running on Kubernetnes.